lib/kms/wrapper.js

```diff
@@ -111,6 +111,25 @@ export default class KMS {
         }
     }
 
+    /**
+     *
+     * @param {string} bucketKeyId - the Id of the bucket key
+     * @param {object} log - logger object
+     * @param {function} cb - callback
+     * @returns {undefined}
+     * @callback called with (err)
+     */
+    static destroyBucketKey(bucketKeyId, log, cb) {
+        log.debug('deleting bucket key', { bucketKeyId });
+        client.destroyBucketKey(bucketKeyId, log, err => {
+            if (err) {
+                log.warn('error from kms', { implName, error: err });
```

Comment:
This should be error level to dump the logs in case of error. Also please add method to the log.